lib/index.js

```diff
@@ -6,6 +7,7 @@ import Redis from './redis'
 import S3 from './s3'
```

Comment:
`import { concat, merge } from 'ramda'` instead of importing all ramda functions